### PR TITLE
Release operator v0.2.0

### PR DESCRIPTION
**Pull request by bcmendoza**

### CODE REVIEW

File: scripts/cli [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 tarball="greymatter-v4.0.1-linux-amd64.tar.gz"
-wget --user=$USERNAME --password=$PASSWORD -q -c "https://nexus.greymatter.io/repository/raw/development/gm-cli/${tarball}"
+wget --user=$USERNAME --password=$PASSWORD -q -c "https://nexus.greymatter.io/repository/raw/release/gm-cli/${tarball}"
```
Comment (dontlaugh) [inline scripts/cli:4]: I wish this was in `scripts/setup` but that's for another time.

**Review by dontlaugh — APPROVED** [review]

**Review by dontlaugh — CHANGES_REQUESTED** [review]
check the CI failures

File: scripts/cli (hunk 1 above)
Comment (bcmendoza) [reply scripts/cli:4]: @dontlaugh We need https://nexus.greymatter.io/repository/raw/release/gm-cli/greymatter-v4.0.1-linux-amd64.tar.gz to be available for this to pass.

File: pkg/version/versions/1.7.cue [hunk 2]
```diff
@@ -1,28 +1,28 @@
 proxy: {
-  image: "docker.greymatter.io/development/gm-proxy:1.7.0-rc.4"
+  image: "docker.greymatter.io/release/gm-proxy:1.7.0"
```
Comment (bcmendoza) [inline pkg/version/versions/1.7.cue:2]: @dontlaugh All these images also need to be available. I can help with this process

**Review by dontlaugh — APPROVED** [review]

**Pull request merged**